pipelines/postprocess_scoring.R

```diff
@@ -329,54 +366,41 @@ main <- function(path_to_scores,
       x = "model"
     )
 
-  wis_comp_by_model_save_path <- get_save_path(
-    "wis_components_by_model",
-    ext = "png"
-  )
   ggsave(
-    wis_comp_by_model_save_path,
+    get_save_path(
+      "wis_components_by_model"
+    ),
     wis_components_by_model
   )
 
-  message("Plotting relative WIS across dates by location")
-  rel_wis_by_location <- relative_wis_by_location(
-    quantile_scores
+  message("Plotting relative WIS across dates by location and horizon")
+  rel_wis_by_location_horizon <- relative_wis_by_location(
+    score_tables$horizon__location
   )
 
-  rel_wis_by_location_save_path <- get_save_path(
-    "relative_wis_by_location"
+  ggsave(
+    get_save_path(
+      "relative_wis_by_location_horizon"
+    ),
+    rel_wis_by_location_horizon,
+    height = 10,
+    width = 8
+  )
+  message("Plotting relative WIS across dates by location and horizon")
+  rel_wis_by_location <- relative_wis_by_location(
+    score_tables$location |> mutate(horizon = "all horizons")
   )
 
-  ggsave(rel_wis_by_location_save_path,
+  ggsave(
+    get_save_path(
+      "relative_wis_by_location"
+    ),
     rel_wis_by_location,
     height = 10,
     width = 4
   )
 
-  message("Making tables...")
-  table_all <- summarised_scoring_table(
-    quantile_scores,
-    scale = "log"
-  )
-
-  table_all_save_path <- get_save_path(
-    "overall_scores",
-    ext = "tsv"
-  )
-  readr::write_tsv(table_all, table_all_save_path)
-
-
-  table_locs <- location_summary_table(quantile_scores,
-    scale = "log"
-  )
-
-  table_locs_save_path <- get_save_path(
-    "scores_by_location",
-    ext = "tsv"
-  )
-  readr::write_tsv(table_locs, table_locs_save_path)
-
-  message("Done with score postprocessing.")
+  message("Done with scoring postprocessing.")
 }
```

Comment:
For saving a bunch of figures, I like the pattern of building a table with the figures, file names, dimensions, etc and  `pwalk`ing, but this is fine.